tests/test_gpu_openmp.py

```diff
@@ -263,3 +309,71 @@ def test_iso_ac(self):
         op(time=time_range.num-1, dt=dt)
 
         assert np.isclose(norm(rec), 490.55, atol=1e-2, rtol=0)
+
+    @pytest.mark.parallel(mode=1)
```

Comment:
Why the `mode=1`?

Reply:
As we discussed on Slack, it was a mistake. I'll change that for 2 and 4 MPI ranks.